src/backend/core/api/viewsets.py

```diff
@@ -376,3 +377,38 @@ def perform_update(self, serializer):
                 raise exceptions.ValidationError({"role": message})
 
         serializer.save()
+
+
+class InvitationViewset(
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet,
+):
+    """API ViewSet for user invitations to team.
+
+    POST /api/v1.0/teams/<team_id>/invitations/ with expected data:
+        - email: str
+        - role: str [owner|admin|member]
+        - issuer : User, automatically added from user making query, if allowed
+        - team : Team, automatically added from requested URI
+        Return newly created invitation
+    """
+
+    lookup_field = "id"
+    pagination_class = Pagination
+    permission_classes = [permissions.AccessPermission]
+    queryset = models.Invitation.objects.all().select_related("team")
+    serializer_class = serializers.InvitationSerializer
+
+    def perform_create(self, serializer, *args, **kwargs):
+        """Verify the issuer is allowed to send invitations to this team."""
+        user = self.request.user
+        team = get_object_or_404(models.Team, pk=self.kwargs["team_id"])
+
+        # check user is team's owner or administrator
+        if not team.get_abilities(user)["manage_accesses"]:
```

Comment:
Maybe better placed in a permission which you can add to the "permissions" attribute above like:
```python
permission_classes = [permissions.AccessPermission, permissions.RelatedTeamAccessPermission]
```

Reply:
Done ! Took me a hot minute, I couldn't remember how permissions classes worked.